### PR TITLE
netdata: detect if pipe exists

### PR DESCRIPTION
**Pull request by Stzx**
###### Description of changes

```
uv_pipe_connect(): no such file or directory
Make sure the netdata service is running.
```

Often the pipe is not actually initialized when the script begins execution.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/modules/services/monitoring/netdata.nix [hunk 1]
```diff
@@ -213,6 +213,7 @@ in {
         ++ lib.optional cfg.python.enable (pkgs.python3.withPackages cfg.python.extraPackages)
         ++ lib.optional config.virtualisation.libvirtd.enable (config.virtualisation.libvirtd.package);
       environment = {
+        NETDATA_PIPENAME = "/tmp/netdata-ipc";
```
Comment (RaitoBezarius) [inline nixos/modules/services/monitoring/netdata.nix:216]: Are you sure the pipe is there?

Comment (Stzx) [reply nixos/modules/services/monitoring/netdata.nix:216]: https://github.com/netdata/netdata/blob/2c305f3bac39a5e404f988921f1587fa67dd12b5/daemon/pipename.c#L8

Comment (RaitoBezarius) [reply nixos/modules/services/monitoring/netdata.nix:216]: How did you test this PR?

Comment (Stzx) [reply nixos/modules/services/monitoring/netdata.nix:216]: > How did you test this PR?
```shell
while [ ! -e "$NETDATA_PIPENAME" ] || [ "$(${cfg.package}/bin/netdatacli ping)" != "pong" ]; do
  if [ -e "$NETDATA_PIPENAME" ]; then echo "$NETDATA_PIPENAME"; fi
  sleep 0.5
done
```

**Pull request closed without merging**